modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -1238,6 +1238,12 @@ definition {
 	macro publishObjectViaUI {
 		Button.clickPublish();
 
+		Alert.viewSuccessMessage();
+
+		WaitForElementNotPresent(locator1 = "Message#SUCCESS");
+
+		WaitForPageLoad();
+
 		WaitForLiferayEvent.initializeLiferayEventLog();
```

Comment:
I don't think this step is necessary. It might don't affect, but sometimes `WaitForLiferayEvent` can through errors and break the tests.

Reply:
It was added before, probably because it was needed to the object actually be published, but since you added those new steps, we might be able to remove it.

Reply:
I just tested without it, and you are right! We don't need `WaitForLiferayEvent`. I will change that and send the updated code.